var/spack/repos/builtin/packages/mlpack/package.py

```diff
@@ -65,6 +65,8 @@ class Mlpack(CMakePackage):
         # ref: src/mlpack/bindings/python/PythonInstall.cmake
         depends_on("py-pip")
         depends_on("py-wheel")
+        # ref: src/mlpack/bindings/python/setup.py.in
+        depends_on("py-setuptools")
```

Comment:
Are the dep types correct here?

Reply:
Thanks. Dep type build is sufficient:
```suggestion
        depends_on("py-setuptools", type="build")
```